administrator/components/com_media/models/api.php

```diff
@@ -38,85 +46,126 @@ public function __construct($config = array())
 	{
 		parent::__construct($config);
 
+		if (!isset($config['providers']))
+		{
+			$config['providers'] = Joomla\CMS\Plugin\PluginHelper::getPlugin('filesystem');
```

Comment:
This assignment is not needed at all. It makes sense when you want to inject the providers trough the config.